lcls2-cc-lib/Library/POUs/FB_L2SI_Flowmeter.TcPOU

```diff
@@ -1,22 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.12">
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
   <POU Name="FB_L2SI_Flowmeter" Id="{f4fe6025-68b2-43d4-8493-5a0f33999a1a}" SpecialFunc="None">
-    <Declaration><![CDATA[FUNCTION_BLOCK FB_L2SI_Flowmeter
-VAR
-    {attribute 'pytmc' := '
-        pv: MA
-        io: input
-    '}
-    fRaw AT %I*: INT;
-
-    {attribute 'pytmc' := '
-        pv: FLOW
-        io: input
-    '}
-    fFlowRate: LREAL;
+    <Declaration><![CDATA[FUNCTION_BLOCK FB_L2SI_Flowmeter Extends FB_AnalogInput
+VAR_INPUT
+    fResolution : LREAL := 0.1;
+    fOffset : LREAL;
 END_VAR]]></Declaration>
     <Implementation>
-      <ST><![CDATA[]]></ST>
+      <ST><![CDATA[SUPER^(iTermBits:=12, fTermMax:=60, fTermMin:=0);
```

Comment:
What are you thoughts on inheritance in TwinCAT? In the past I've found it super clunky to debug. Was it OK in this case?

Reply:
For the most part my personal opinion is that its to be avoided. However, in a 1 deep inheritance tree with no multi inheritance it seems appropriate. I ran this code online, when I opened the FB the variables appeared normal without any obvious complications.